curvefs/src/client/warmup/warmup_manager.cpp

```diff
@@ -173,59 +174,35 @@ void WarmupManagerS3Impl::LookPath(fuse_ino_t key, std::string file) {
         VLOG(9) << "empty path";
         return;
     }
-    bool isRoot = false;
-    if (file == "/") {
-        splitPath.push_back(file);
-        isRoot = true;
-    } else {
-        curve::common::AddSplitStringToResult(file, "/", &splitPath);
+
+    if (!curve::common::StringStartWith(file, "/")) {
+        LOG(ERROR) << fmt::format("{} isn't absolute path", file);
+        file.erase(0, 1);
```

Comment:
What if it is not an absolute path? And what is the purpose of `file.erase(0, 1); `

Reply:
> What if it is not an absolute path? And what is the purpose of `file.erase(0, 1); `

If it is not an absolute path, the warm-up task will be skipped.
There should be a problem with the latter one, it has been fixed